### ISSUE TITLE
Moe-theme-switcher breaks theme background color

### ISSUE THREAD

**Issue by Aaronzinhoo** [kuanyui/moe-theme.el#108]
Hi, I have recently run into a problem when using the dev branch. That causes the moe-theme to be loaded twice, (this makes sense since the switcher loads it within its file as well) and causes the moe-theme-dark and light backgrounds to be merged.  

This is my configuration for moe-theme. I notice when I remove ```(require 'moe-theme-switcher)``` from my config, the dark theme loads properly. The first image is in regards to the overlapping themes displayed. The second image is to show moe-theme-switcher is loading twice.
```
(use-package moe-theme
  :straight (:type git :host github :repo "kuanyui/moe-theme.el" :branch "dev")
  :config
  (require 'moe-theme-switcher)
  (setq moe-theme-highlight-buffer-id t)
  (powerline-moe-theme))
```

![Screenshot (4)](https://user-images.githubusercontent.com/26849064/94266231-b0329d00-feee-11ea-8705-f324c9ec0983.png)

![Screenshot (5)](https://user-images.githubusercontent.com/26849064/94266678-54b4df00-feef-11ea-92b2-c0b5e288ec7b.png)

I really love your work on theme and hope there is a way to solve this!

**Comment by Aaronzinhoo** [comment]
After inspecting the code, these are actually two separate issues. I am testing some small changes and will make a PR if the tests work out.

**Comment by Aaronzinhoo** [comment]
Fixed in PR [#109](https://github.com/kuanyui/moe-theme.el/pull/109)
